compiler/rustc_lint/messages.ftl

```diff
@@ -192,6 +192,12 @@ lint_redundant_semicolons =
         *[false] this semicolon
     }
 
+lint_unnecessary_send_constraint = constraining a reference to `Send` is meaningless
+    .suggestion = {$only_trait ->
+        [true] replace this with `std::any::Any`
```

Comment:
Replacing by `Any` is a very strange suggestion. Why would `type_id` machinery be necessary? Can't the `'static` bound cause a problem?

Reply:
I wasn't sure what else to put, this is for the scenario where `Send` is the only trait specified (so removing it would be invalid)